modules/text/include/opencv2/text/erfilter.hpp

```diff
@@ -227,6 +227,43 @@ features: hole area ratio, convex hull ratio, and number of outer inflexion poin
 CV_EXPORTS_W Ptr<ERFilter> createERFilterNM2(const Ptr<ERFilter::Callback>& cb,
                                                   float minProbability = (float)0.3);
 
+/** @brief Create an Extremal Region Filter for the 1st stage classifier of N&M algorithm [Neumann12].
+
+@param  filename :   The path to the classifier XML, e.g. /path/to/cpp/trained_classifierNM1.xml
+@param  thresholdDelta :   Threshold step in subsequent thresholds when extracting the component tree
+@param  minArea :   The minimum area (% of image size) allowed for retreived ER's
+@param  maxArea :   The maximum area (% of image size) allowed for retreived ER's
+@param  minProbability :   The minimum probability P(er|character) allowed for retreived ER's
+@param  nonMaxSuppression :   Whenever non-maximum suppression is done over the branch probabilities
+@param  minProbabilityDiff :   The minimum probability difference between local maxima and local minima ERs
+
+The component tree of the image is extracted by a threshold increased step by step from 0 to 255,
+incrementally computable descriptors (aspect_ratio, compactness, number of holes, and number of
+horizontal crossings) are computed for each ER and used as features for a classifier which estimates
+the class-conditional probability P(er|character). The value of P(er|character) is tracked using the
+inclusion relation of ER across all thresholds and only the ERs which correspond to local maximum of
+the probability P(er|character) are selected (if the local maximum of the probability is above a
+global limit pmin and the difference between local maximum and local minimum is greater than
+minProbabilityDiff).
+ */
+CV_EXPORTS_W  Ptr<ERFilter> createERFilterNM1(const String& filename,
+                                                  int thresholdDelta = 1, float minArea = (float)0.00025,
+                                                  float maxArea = (float)0.13, float minProbability = (float)0.4,
+                                                  bool nonMaxSuppression = true,
+                                                  float minProbabilityDiff = (float)0.1);
+
+/** @brief Create an Extremal Region Filter for the 2nd stage classifier of N&M algorithm [Neumann12].
+
+@param  filename :   The path to the classifier XML, e.g. /path/to/cpp/trained_classifierNM2.xml
+@param  minProbability :   The minimum probability P(er|character) allowed for retreived ER's
+
+In the second stage, the ERs that passed the first stage are classified into character and
```

Comment:
Hi sovrasov,

I'm not very strong with doxygen - do you mind sharing some examples how I need to document such cases?